website/docs/reference/commands/init.md

```diff
@@ -17,10 +17,19 @@ Then, it will:
 - Create a new folder with your project name and sample files, enough to get you started with dbt
 - Create a connection profile on your local machine. The default location is `~/.dbt/profiles.yml`. Read more in [configuring your profile](/docs/core/connect-data-platform/connection-profiles).
 
+<VersionBlock firstVersion="1.7">
+
+Use the `--profile` flag with  `dbt init` to initialize the project with an existing `profiles.yml` as the `profile:` key instead of creating a new one.
```

Comment:
sugg a reword to simplify. i had to reread the original a few times but may be just me

```suggestion
When using `dbt init` to set up your project, include the `--profile flag` to specify an existing `profiles.yml` as the `profile:` key to use instead of creating a new one.
```

Reply:
I like these changes. I think the preferred wording is "initialize" over "set up", so I'll keep that bit and roll in the remainder